app/themes/index.global.scss

```diff
@@ -114,78 +114,80 @@ button {
 }
 
 // Electrolize
+// TODO: Not used anymore, delete when doing refactoring
 
 @font-face {
   font-family: Electrolize, 'Electrolize-Regular';
   src: url("../assets/fonts/Electrolize-Regular.ttf") format("truetype");
 }
 
 // Montserrat
+// TODO: Not used anymore, delete when doing refactoring
 
 @font-face {
   font-family: 'Montserrat-Medium';
   src: url("../assets/fonts/Montserrat-Medium.ttf") format("truetype");
 }
 
-// Rubik
+// Helvetica
+
+@font-face {
+  font-family: 'Helvetica-Regular';
```

Comment:
@Marta-Ryzhok do we really need this one?

Reply:
cc @SebastienGllmt